docker-compose-htsget.yml

```diff
@@ -14,14 +14,14 @@ services:
     volumes:
       # If you want to start the docker compose without tls, remove the line below,
       # remove the starter-kit-storage-and-interfaces_shared volume from the bottom
-      # of the file and the relevant configuration from the config-htsget/config.json 
+      # of the file and the relevant configuration from the config-htsget/examples/config.json
       # file, which are the lines for serverCert and serverKey.
       # The TLS version assumes that this compose file is running along with the
       # storage and interfaces.
       # If you want to run the htsget with TLS but without the storage and interfaces,
       # add the certificates in the volume below
       - starter-kit-storage-and-interfaces_shared:/shared
-      - ./config-htsget:/config
+      - ./config-htsget/examples:/config
```

Comment:
It seems strange that the compose file refers to the `config-htsget/examples` subdirectory and not to files in `config-htsget`. I thought placing the example configurations in a `examples` subdirectory meant the _real_, _live_, configuration would be read from the parent directory. I.e., the user should copy the example files out of the `example` directory and then modify those copies.

The other common way to do this is to keep the example configurations in files with a `.example` filename suffix, and let the user copy these to the real configuration file, i.e. copy `config.json.example` to `config.json` and then modify that file.

Reply:
I agree with what @kusalananda suggested.

Reply:
done